data/avgi/avgi sales.txt

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+shipyard "Avgi Developer"
+	"Akoustiki"
+	"Atenuatia"
+	"Avlaki"
+	"Diaspersi"
+	"Difraktos"
+	"Entasi"
+	"Frequentia"
+	"Harmonikos"
+	"Interferos"
+	"Koryfi"
+	"Melodikos"
+	"Modified Tachytia"
+	"Optikon"
+	"Phononos"
+	"Photikon"
+	"Polari"
+	"Prismaios"
+	"Refraktos"
+	"Sonikis"
+	"Tremoros"
+	"Undsyni"
+	"Undulon"
+	"Vibratia"
+
+shipyard "Avgi Civilian Basics"
+	"Koryfi"
+	"Avlaki"
+	"Frequentia"
```

Comment:
The Frequentia runs on a Spark, and is evidently available in basically every shipyard. Is it worth restricting the Z-330 Spark, XY-180, Inductive Extractor, and Remass Injectors to the Twilight Guard license? I am aware it's a bit of a flavor thing.